docs/advanced/secure-setup.md

```diff
@@ -0,0 +1,314 @@
+# <img class="dcr-icon" src="/img/dcr-icons/LockEye.svg" /> Secure Setup
+
+Last updated for CLI release v{{ cliversion }}.
+
+---
+
+This document will guide how to setup a two system method for holding your Decred securely.
+Some of the commands given will be specific to Debian based systems, but the principles/tools can be applied to almost any OS.
+
+**Please note that this setup requires two computers.**
+
+Although focused on Decred and Raspberry Pi, this guide can also be used for
+more general hardware devices and almost any cryptocurrency.
+
+---
+
+## Scope and Limitations
+
+The scope of this guide is to secure your cryptocurrency holdings from common malware and light threats.
+If you are a normal cryptocurrency holder and follow this guide exactly, it should give a good level of security.
+
+The guide assumes some general technical knowledge and that you can fix any hitches you come across.
+
+**This setup will not protect you against a state-level attacker or a persistent threat.**
+
+[General Security](general-security.md) offers more techniques to keep your system secure.
+
+---
+
+## The Setup
+
+This will be a multi-system setup.
+
+![Network diagram](/img/secure-setup.png)
+
+* A laptop or desktop that will be running the dcrd node, referred to as `Computer-A`.
+* A secure system that will be running the wallet, referred as `Wallet-B`.
+  This guide will use a Raspberry Pi 4, however it could also be a laptop or desktop.
+  This system should be connected to an external monitor and keyboard, and it
+  should also connect to the local network via ethernet.
+* All details/commands that come within curly braces `{}` cannot be copy-pasted,
+  you will need to remove the curly braces and edit them to suit your setup.
+
+### Benefits of this Setup
+
+* `Wallet-B` is kept offline and turned off most of the time.
+  This greatly reduces attack surface/opportunity.
+* An attacker would have to compromise dcrd on `Computer-A` first, and then find a
+  way to jump into `Wallet-B` using the RPC to compromise and exfiltrate data.
+  This is highly unlikely and difficult to pull off.
+* dcrd on `Computer-A` can be used for other purposes in your local network (eg. other wallets, DEX, etc),
+  so you don't need to maintain multiple dcrd instances within your local network.
```

Comment:
Suggest adding two more bullet points as additional, benefits / explanation to this setup:

* Even though `Wallet B` is kept offline, it’s still possible for it to participate in staking by using a VSP.
* Advanced readers can extrapolate from this guide and make `Wallet B` a “hot” wallet so they can Solo Stake, but that’s significantly more vulnerable to attacks and outside the scope of this guide.